src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -320,9 +323,8 @@ class ImageBaseViewer extends BaseViewer {
         // eslint-disable-next-line
         console.error(err);
 
-        // Display a generic error message but log the real one
-        const error = new PreviewError(ERROR_CODE.CONTENT_DOWNLOAD, __('error_refresh'), {}, err.message);
-        super.handleDownloadError(error, imgUrl);
+        const genericDownloadError = new PreviewError(ERROR_CODE.CONTENT_DOWNLOAD, __('error_refresh'));
```

Comment:
do we not need the extra params passed into the `PreviewError` constructor?

Reply:
Nope, they're optional past the first param (error code)

Reply:
true, but this one should stay since it was there previously. I had modified this function before but it's not necessary anymore.